python/ray/tune/trial_runner.py

```diff
@@ -673,50 +668,69 @@ def step(self):
         # Update status of staged placement groups
         self.trial_executor.stage_and_update_status(self._live_trials)
```

Comment:
This can be moved into `step()`

Reply:
Good point. Make it one less API to expose for TrialExecutor.